src/lib.rs

```diff
@@ -61,6 +61,26 @@ trait GuardedTransmute<D> {
 #[macro_export]
 macro_rules! impl_guarded_transmute {
     ($src_type:ty, $dst_type:ty) => {
+        impl_guarded_transmute!(noderefs $src_type, $dst_type);
```

Comment:
Would be nice to have some comments about new different `impl_guarded_transmute` modes and when to use which one